src/cli/README_DATASET.md

```diff
@@ -513,6 +513,10 @@ Set security policy.
 - r: Thread 1.x Routers are enabled.
 - c: External Commissioner authentication is allowed using PSKc.
 - b: Thread 1.x Beacons are enabled.
+- C: Thread 1.2 Commercial Commissioning is enabled.
+- e: Thread 1.2 Autonomous Enrollment is enabled.
+- p: Thread 1.2 Network Master Key Provisioning is enabled.
+- R: Non-CCM routers are allowed in Thread 1.2 CCM networks.
 
 ```bash
 > dataset securitypolicy 672 onrcb
```

Comment:
@jwhui I didn't update the example command to include the new flags, because those new flags are about CCM and we currently don't have CCM implementation here.